apps/extension/src/routes/popup/approval/origin.tsx

```diff
@@ -83,7 +83,7 @@ export const OriginApproval = () => {
                   </div>
                 </div>
                 <div className='z-30 flex h-11 w-full items-center overflow-x-scroll rounded-lg bg-background p-2 text-muted-foreground [scrollbar-color:red_red]'>
-                  <div className='mx-auto items-center p-2 text-center leading-[0.8em] [background-clip:content-box] [background-image:repeating-linear-gradient(45deg,red,black_15px)] first-line:[background-color:black]'>
```

Comment:
The linter was complaining about `first-line:[background-color:black]` not being valid, however, it looks like most of these aren't used anway

Reply:
this was actually an important style that is typically invisible but creates a visual indication if the line breaks

Reply:
also, in this case the linter is incorrect